__tests__/app.test.js

```diff
@@ -196,26 +176,84 @@ describe("/api/articles/:article_id/comments", () => {
         });
       });
   });
+
   test("Should return sorted comments in desc order by date.", () => {
     return request(app)
       .get("/api/articles/1/comments")
       .expect(200)
       .then(({ body }) => {
         const comments = body.comments;
+        
+
+        expect(comments).toBeSortedBy('created_at', {
+          descending: true,
+          coerce: true,
+        });
+      });
+  });
+
+  test("POST: 201 adds a new comment by article_id, and returns the comment.", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "Interesting text",
+    };
+
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const comment = body.comment;
+
+        expect(typeof comment).toBe("object");
```

Comment:
try using .objectContaining and .toMatchObject instead of checking the types on all these properties